src/components/Home/HomeHeader.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { AppBar, Toolbar, Typography, Button } from '@mui/material';
+import { Add, AccountCircle } from '@mui/icons-material';
+
+const HomeHeader = () => {
```

Comment:
I'm not too sure if this header will be used repeatedly for the rest of the website. Can check with @ShenyiCui . If it is, it can be in a component folder of its own, similar to footer.